src/main/java/mate/academy/spring/model/Order.java

```diff
@@ -15,6 +17,8 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @OneToMany
+    @JoinTable(joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
```

Comment:
add table name "orders_tickets" here

also, add `@Table(name = "orders")` above the class